platform/sync-installer/sync-installer-impl/src/main/java/org/codice/ddf/sync/installer/impl/SynchronizedInstallerImpl.java

```diff
@@ -62,11 +62,11 @@
 
   private static final String SERVICE_PID_FILTER = "(" + Constants.SERVICE_PID + "=%s)";
 
-  private static final long DEFAULT_MAX_FEATURE_WAIT = TimeUnit.MINUTES.toMillis(10);
+  private static final long DEFAULT_MAX_FEATURE_WAIT = TimeUnit.MINUTES.toMillis(2);
```

Comment:
❓ Why did we change this? This could negatively impact the installation of the production system on slow systems.

Reply:
OH CRUD! My bad.